### PR TITLE
feat: unban user in community

### LINKED ISSUE #2724: Expose Community BanList to Nim & unban community members
# Problem
Adding unban community member functionality
Expose Community BanList to Nim

# Implementation
Changes in protocol/communities/community.go in order to use info in Nim

Needed by https://github.com/status-im/status-desktop/issues/5979

### PR DESCRIPTION
**Pull request by endulab**
Issue #2724

Unban user in community.
Expose unban functionality to Nim.

### CODE REVIEW

File: protocol/messenger_communities.go [hunk 1]
```diff
@@ -979,6 +979,23 @@ func (m *Messenger) SendKeyExchangeMessage(communityID []byte, pubkeys []*ecdsa.
 	return nil
 }
 
+func (m *Messenger) UnbanUserFromCommunity(request *requests.UnbanUserFromCommunity) (*MessengerResponse, error) {
+	community, err := m.communitiesManager.UnbanUserFromCommunity(request)
+	if err != nil {
+		return nil, err
+	}
+
+	// TODO generate new encryption key
```
Comment (endulab) [inline protocol/messenger_communities.go]: @cammellos I am not sure this is needed.

Comment (cammellos) [reply protocol/messenger_communities.go]: No need to rekey when unbanning, so we can fully remove the commented out code

**Review by cammellos — COMMENTED** [review]
Looks good, I would add a test testing the new functionality and making sure that the updated community is returned to the client, otherwise looks good!

**Review by cammellos — APPROVED** [review]

**Review by osmaczko — APPROVED** [review]
:+1:

**Pull request merged**